app/models/form_profile/va1990e.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+class FormProfile::VA1990e < FormProfile
+  def prefill(user)
+    super
+  end
+
+  def metadata
+    {
+      version: 0,
+      prefill: true,
+      returnUrl: '/1990e/applicant/information'
```

Comment:
On the front end we ignore the `returnUrl` when prefilling, I think, but all the edu form profiles should have the url changed to remove the form id (e.g. this should be `/applicant/information`).

This changed a couple weeks ago when we refactored how the forms are structured on the front end.

Reply:
Sounds good! I'll fix this up.